setting/console_setting/validation.go

```
@@ -24,12 +26,16 @@ var (
 
 func parseJSONArray(jsonStr string, typeName string) ([]map[string]interface{}, error) {
 	var list []map[string]interface{}
-	if err := json.Unmarshal([]byte(jsonStr), &list); err != nil {
+	if err := common.UnmarshalJsonStr(jsonStr, &list); err != nil {
 		return nil, fmt.Errorf("%s格式错误：%s", typeName, err.Error())
 	}
 	return list, nil
 }
 
+func exceedsMaxCharacters(s string, max int) bool {
```

golangci-lint (2.12.2)

[error] param max has same name as predeclared identifier

(predeclared)